### DISCUSSION TITLE
Use popover for Library items

### DISCUSSION THREAD

**Discussion by pravdomil** [keiwando/toonsquid#255] [Ideas]
it is not clear what clicking on library item will do (the title only says Library)

many times I just want to edit a clip and I end up accidentally inserting it into current scene

so I would show a popup that gives me a list of options, so I explicitly choose an action

this is somehow related how illustrator/animate/after effects work, clicking on symbol/comp opens it

<img width="865" alt="image" src="https://user-images.githubusercontent.com/2387356/230777832-41cf559d-603b-4f52-8667-47537af0edc6.png">

> and as always I can code it for free if you want

**Comment by Cat-Butler** [comment]
As you have probably already discovered by now (answer added for the benefit of later readers of this discussion):

![LibraryMenu-sml](https://user-images.githubusercontent.com/25494772/233553859-7365a09f-b48c-4749-9a28-4b982ccf5230.jpg)

**Reply by pravdomil** [reply]
but you need to long press in one area and tap edit in another area, my solution is two quick taps next to each other

**Reply by Cat-Butler** [reply]
Good point! 
I would be happy with your suggested change to speed things up, and it would be consistent with behaviour of the brush and layer areas.

**Reply by pravdomil** [reply]
exactly
